release_task.rb

```diff
@@ -143,13 +155,17 @@ def define_version_update_task
     namespace :version do
       desc "Update version"
       task :update do
-        latest_release = GitHubClient.new(@product, @product).latest_release
-        # "Groonga 14.1.1 - 2024-12-03" or "Mroonga 14.11 - 2024-12-03"
-        release_name = latest_release["name"]
-        # "14.1.1" or "14.11"
-        latest_version = release_name[/\d+(\.\d+){1,2}/, 0]
-        # "2024-12-03"
-        latest_release_date = release_name[/\d+-\d+-\d+/, 0]
+        github_client = GitHubClient.new(@product, @product)
+        latest_tag = github_client.latest_tag
+        tag_commit = github_client.fetch_by(latest_tag["commit"]["url"])
+        # "v14.1.3"(Groonga), "v14.14"(Mroonga) or "3.2.5"(PGroonga)
+        release_tag_name = latest_tag["name"]
+        # "14.1.3", "14.14" or "3.2.5"
+        latest_version = release_tag_name[/\d+(\.\d+){1,2}/, 0]
+        # "2025-01-29T03:03:15Z(UTC)" -> "2025-01-30(JST)"
```

Comment:
```suggestion
        # "2025-01-29T03:03:15Z(UTC)" -> "2025-01-29(JST)"
```

```
irb(main):003:0> Time.parse("2025-01-29T03:03:15Z").getlocal("+09:00")
=> 2025-01-29 12:03:15 +0900
```

日付が変わる例ならばUTCの17時以降が良いです。

```
irb(main):007:0> Time.parse("2025-01-29T17:03:15Z").getlocal("+09:00")
=> 2025-01-30 02:03:15 +0900
```

Reply:
17時じゃないか15時か。

Reply:
15時だと思って書いていたのに、全然3時でしたね。。。直します。

Reply:
fix: [ee7105a](https://github.com/groonga/groonga.org/pull/95/commits/ee7105afaed04924dd2df7e1a3092a248e8754dd) 15時に例を変更しました。